src/server/socket.js

```diff
@@ -1,103 +1,204 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 
+/*
+ * This module implements communication protocol as described
+ */
+
 var log = require('./log'),
     config = require('./config'),
     livereload = require('./live-reload/live-reload-server'),
-    telemetry = require('./telemetry-helper');
+    telemetry = require('./telemetry-helper'),
+    q = require('q');
```

Comment:
The variable name should be with uppercase, for consistency with other usages and since `Q` is the global when no using amd loader.